CI failing with type error in tabs component

```
../src/components/layout/tabs/index.tsx:28:11 - error TS2322: Type 'Element[] | null | undefined' is not assignable to type 'ReactNode[]'.
  Type 'undefined' is not assignable to type 'ReactNode[]'.
28     const tabs: React.ReactNode[] = React.Children.map(self.props.children, (child: any, i: number) => {
             ~~~~
```

--- a/src/components/layout/tabs/index.tsx
+++ b/src/components/layout/tabs/index.tsx
@@ -25,7 +25,7 @@ export default class Tabs extends React.PureComponent<Props, {}> {
       }
     }
 
-    const tabs: React.ReactNode[] = React.Children.map(self.props.children, (child: any, i: number) => {
+    const tabs = React.Children.map(self.props.children, (child: any, i: number) => {
       if (child.props['data-key'] === undefined) {
         return null
       }

